application/src/main/java/com/sanction/lightning/resources/FacebookResource.java

```diff
@@ -34,7 +37,7 @@ public FacebookResource(ThunderClient thunderClient, FacebookApplicationKey appl
   }
 
   /**
-   * Fetches a user object containing user information.
+   * Fetches a FacebookUser object containing user information.
    * @param key
    * @param username
```

Comment:
I missed this on the last PR, can you update this javadoc so that the `@param` tags are filled?